src/services/roomservice/index.ts

```diff
@@ -14,18 +14,25 @@ const transformSpace = (spaceData: Fi2Space): Room => {
   return room
 }
 
-const getRooms = async (rentalId?: string): Promise<Room[]> => {
+const getRooms = async (rentalId?: string, isShared?: boolean): Promise<Room[]> => {
   try {
+    const filterIfShared = isShared ? `;fi2space_common:'${isShared}'` : ''
```

Comment:
This way we can't filter out rooms that are not shared though 😬  (if isShared is false, then this is an empty string)